cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DocumentRevision.java

```diff
@@ -242,7 +241,7 @@ long getParent() {
      *
      * @return  the internal numeric ID of this document revision.
      */
-    long getInternalNumericId(){
+    public long getInternalNumericId(){
```

Comment:
I'm opposed to this particular change because this class is API public. I think the internal numeric ID exposes way to much implementation detail to be part of a public API.